saleor/csv/notifications.py

```diff
@@ -0,0 +1,40 @@
+from typing import TYPE_CHECKING
+
+from ..core.notify_events import NotifyEventType
+from ..core.utils import build_absolute_uri
+from ..plugins.manager import get_plugins_manager
+from . import events
+
+if TYPE_CHECKING:
+    from .models import ExportFile
+
+
+def get_default_export_payload(export_file: "ExportFile") -> dict:
+    user_id = export_file.user.id if export_file.user else None
+    user_email = export_file.user.email if export_file.user else None
+    app_id = export_file.app.id if export_file.app else None
+    return {
+        "user_id": user_id,
+        "user_email": user_email,
+        "app_id": app_id,
+        "status": export_file.status,
+        "message": export_file.message,
+        "created_at": export_file.created_at,
+        "updated_at": export_file.updated_at,
+    }
+
+
+def send_email_with_link_to_download_file(export_file: "ExportFile"):
```

Comment:
A little naming inconsistency - here we use "email" in the name, while the module is called "notification". If I understand we can implement all sort of notification and emails are just one type of them, is that correct? Then I would call these function in the same manner like `send_export_download_link_notification` or `notify_export_download_link`.

Reply:
Also we should add some docstrings.